app/src/main/java/com/keylesspalace/tusky/components/followedtags/FollowedTagsActivity.kt

```diff
@@ -180,6 +184,15 @@ class FollowedTagsActivity :
         return viewModel.searchAutocompleteSuggestions(token)
     }
 
+    override fun view(tagName: String) {
+        startActivity(StatusListActivity.newHashtagIntent(this, tagName))
+    }
+
+    override fun copyTagName(tagName: String) {
+        getSystemService(ClipboardManager::class.java).setPrimaryClip(ClipData.newPlainText(null, "#$tagName"))
+        Toast.makeText(this, getString(R.string.confirmation_hashtag_copied, tagName), Toast.LENGTH_SHORT).show()
```

Comment:
modern Android anyway shows popup when something is copied, I am wondering if we need anything else

Reply:
Huh, it didn't for me (android 11), and we're manually showing a popup in other places, but I can remove it if newer android has us covered

Reply:
From the [documentation](https://developer.android.com/develop/ui/views/touch-and-input/copy-paste#Feedback):

Users expect visual feedback when an app copies content to the clipboard. This is done automatically for users in Android 13 and higher, but it must be manually implemented in prior versions.

Reply:
I moved all the slightly different "copy text to clipboard" implementations to an `Activity` extension that puts the toast behind a version check